src/_includes/home/sections/hero.njk

```diff
@@ -12,13 +12,13 @@
         {{ site.home.hero.body }}
       </p>
       <div class="mt-8 flex flex-wrap gap-4 justify-center md:justify-start">
-        <a class="btn btn-primary w-full sm:w-36" href="{{ site.home.hero.buttons.primary.link }}" rel="noreferrer" target="_blank">
+        <a class="btn btn-primary w-full sm:w-36" href="{{ site.home.hero.buttons.primary.link }}" rel="noreferrer" target="_blank" aria-label="Link to the CS++ Society Page on the clubs and societies portal">
```

Comment:
Seems like this could get out of sync with the link if config was changed. Same comment for the rest of the buttons.

Reply:
Resolved in latest commit.